### DISCUSSION TITLE
updated documentation on how to use this

### DISCUSSION THREAD

**Discussion by DennyFiguerres** [OData/RESTier#732] [General]
i found old docs from way back and no actual current guide on how to use this....
also the repo does not really show that this is ready for dot net core 6 or 7....
but i see that work is beeing done for v8 ....
so can we please at least get some short samples of how to setup for core v7 ?

**Comment by Tiberriver256** [comment]
I think this is a duplicate of #727?

**Reply by DennyFiguerres** [reply]
related for sure..... do you know of any sample of the current use of restier ?  i am looking for any to see if this is worth trying to use.
also trying to see if it will be maintained and stay current.....

**Comment by robertmclaws** [comment]
This project has supported ASP.NET Core on OData V7 for a couple years now. Please look at the following samples:

https://github.com/OData/RESTier/tree/main/src/Microsoft.Restier.Samples.Northwind.AspNet

https://github.com/OData/RESTier/tree/main/src/Microsoft.Restier.Samples.Northwind.AspNetCore

**Comment by robertmclaws** [comment]
Closing as a duplicate of #727
